invenio_github/tasks.py

```diff
@@ -141,3 +143,28 @@ def _get_err_obj(msg):
             u'Error while processing {release}'.format(release=release.model))
     finally:
         db.session.commit()
+
+
+@shared_task(ignore_result=True)
+def refresh_accounts(expiration_threshold):
+    """Refresh stale accounts, avoiding token expiration.
+
+    expiration_threshold = {'weeks': 8}
```

Comment:
I guess leftovers... Actually, you can add a docstring `:param expiration_threshold:` with exactly what should be passed. 

It would be also nice to have a sane default for this, so it can be changed to `expiration_threshold=None` and then do something like `expiration_threshold = expiration_threshold or datetime.timedelta(months=6)` in the task.